src/platforms/web/runtime/modules/dom-props.js

```diff
@@ -20,6 +21,7 @@ function updateDOMProps (oldVnode: VNodeWithData, vnode: VNodeWithData) {
 
   for (key in oldProps) {
     if (!(key in props)) {
+      // TT_TODO: when (how) is this even called
```

Comment:
This line is called when users use object bindings, e.g. `v-bind="obj"`, and the new object has different keys from the old object. For example, on first render `obj` could be `{ id: 'foo' }` and later updated to `{ innerHTML: 'bar' }` - in this case `id` needs to be unset. This can also happen in manually written render functions.

Reply:
Thanks, however I don't think I understand.

The `props` and `oldProps` are domProps from [render function](https://vuejs.org/v2/guide/render-function.html) and `v-bind` binds values from [data property of the Vue component](https://vuejs.org/v2/guide/components.html#data-Must-Be-a-Function). I am still unable to trigger that part of the code, but I was trying to do something like [this codesandbox](https://codesandbox.io/embed/vue-template-dndzq). I also had a look at the original issue [here](https://github.com/vuejs/vue/issues/9650), but their example is different from what I want to see.

The important question is: **Can the code (where the TODO comment is) be called for dangerous properties like `innerHTML`?**

If yes, I need to add integration on this place as well, but I don't wanna include it on places which don't need it.